calcom/cal.com
# Prevent updating app to enabled on seed app store

## What does this PR do?



This PR prevents enabling apps on the seed event if they are disabled

Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change



- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?



- As an admin disable some apps
- Either run `yarn dx` or seed the app store. The apps should stay disabled

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
